tests/Doctrine/Tests_PHP81/Persistence/Reflection/EnumReflectionPropertyTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Doctrine\Tests_PHP81\Persistence\Reflection;
+
+use Doctrine\Persistence\Reflection\EnumReflectionProperty;
+use PHPUnit\Framework\TestCase;
+use ReflectionProperty;
+use ValueError;
+
+class EnumReflectionPropertyTest extends TestCase
+{
+    public function testGetValue(): void
+    {
+        $object = new TypedEnumClass();
+
+        $reflProperty = new EnumReflectionProperty(new ReflectionProperty(TypedEnumClass::class, 'suit'), Suit::class);
+
+        self::assertNull($reflProperty->getValue($object));
+
+        $object->suit = Suit::Clubs;
+
+        self::assertSame('C', $reflProperty->getValue($object));
+
+        $object->suit = null;
+
+        self::assertNull($reflProperty->getValue($object));
```

Comment:
Skipping lines is usually used to group lines together in a block. Here, nothing is grouped, so why even have blank lines?